sonar-fsharp-plugin/src/main/java/org/sonar/plugins/fsharp/FSharpPlugin.java

```diff
@@ -55,7 +55,7 @@
   public static final String FILE_SUFFIXES_KEY = "sonar.fs.file.suffixes";
   public static final String FILE_SUFFIXES_DEFVALUE = ".fs,.fsx,.fsi";
 
-  public static final String FSHARP_WAY_PROFILE = "Sonar way";
+  public static final String FSHARP_WAY_PROFILE = "F# Rules";
```

Comment:
Should the Name indicate, that these are the Plugins default rule set? eg. "F# default rules"

Reply:
Sure